deny.toml

```diff
@@ -48,6 +48,7 @@ allow = [
   "BSD-3-Clause",
   "Unicode-DFS-2016",
   "Unicode-3.0",
+  "MPL-2.0",
```

Comment:
Otherwise:

```
error[rejected]: failed to satisfy license requirements
  ┌─ registry+https://github.com/rust-lang/crates.io-index#option-ext@0.2.0:4:12
  │
4 │ license = "MPL-2.0"
  │            ^^^^^^^
  │            │
  │            license expression retrieved via Cargo.toml `license`
  │            rejected: license is considered copyleft
  │
  = MPL-2.0 - Mozilla Public License 2.0:
  =   - OSI approved
  =   - FSF Free/Libre
  =   - Copyleft
  = option-ext v0.2.0
    └── dirs-sys v0.4.1
        └── dirs v5.0.1
            └── shellexpand v3.1.0
                └── qcs-api-client-common v0.8.4
                    ├── qcs v0.22.0
                    ├── qcs-api-client-grpc v0.8.4
                    │   └── qcs v0.22.0 (*)
                    └── qcs-api-client-openapi v0.9.4
                        └── qcs v0.22.0 (*)
```

Reply:
This is no longer needed after pinning `dirs` to `v4.0.0`